Realm/Realm/Sync/ProgressNotifications/SyncProgress.cs

```diff
@@ -36,14 +39,24 @@ public struct SyncProgress
         /// successfully transferred.
         /// </summary>
         /// <value>The number of transferable bytes.</value>
+        [Obsolete("Not accurate, use ProgressEstimate instead.")]
         public ulong TransferableBytes { get; }
```

Comment:
Since this is going into next-major, we don't need to obsolete them and can just remove the fields instead.

Reply:
Right!